docs/how-to-guides/Interact-with-gnoland.md

```diff
@@ -0,0 +1,94 @@
+---
+id: interact-with-gnoland
+---
+
+# Interact with Gno.land
+
+This tutorial will teach you how to interact with the Gnoland blockchain by creating an account and calling various realms to send transactions on the network.
+
+## Prerequisites
+
+- [Installation](../getting-started/installation.md)
```

Comment:
Where is the file? It is mentioned in multiple documents, but I can't locate it.

Additionally, I recommend changing the link. The term "Installation" is unclear regarding what we intend to install.

Reply:
It's the initial document in `getting-started`. I've renamed it to `Local Setup`, which is what it actually is:
[1e627ef](https://github.com/gnolang/gno/pull/1046/commits/1e627efa1a46ac0cd8ee3c1e65c7f6e2da064455)